CrowdFundingApplication/Views/Shared/_Layout.cshtml

```diff
@@ -3,43 +3,69 @@
-                        <li class="nav-item">
-                            <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
-                        </li>
-                    </ul>
-                </div>
+<body class="bg-light">
+
+    @*NAVBAR*@
+    <nav class="navbar navbar-expand-lg navbar-dark fixed-top">
+        <div class="container text-white">
+            <a class="navbar-brand" asp-area="" asp-controller="Home" asp-action="HomePage">ViewStar</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target=".navbar-collapse" aria-controls="navbarSupportedContent"
+                    aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="navbar-collapse collapse">
+                <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                    <li class="nav-item">
+                        <a class="nav-link" asp-area="" asp-controller="Home" asp-action="Index">Проекты</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" asp-area="" asp-controller="Home" asp-action="Privacy">Контакты</a>
+                    </li>
+                    <li>
+                </ul>
+                <form class="ml-auto d-flex">
+                    <input class="form-control me-2" type="search" placeholder="Поиск" aria-label="Search">
+                    <button class="ml-2 btn btn-dark" type="submit">Поиск</button>
+                </form>
+                <ul class="navbar-nav ml-sm-auto">
+                    <li class="nav-item">
+                        <a class="nav-link" asp-area="" asp-controller="Project" asp-action="CreateProject">Создать проект</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" asp-area="" asp-controller="Profile" asp-action="Index">Профиль</a>
+                    </li>
+                </ul>
             </div>
-        </nav>
-    </header>
+        </div>
+    </nav>
+
     <div class="container">
         <main role="main" class="pb-3">
             @RenderBody()
         </main>
     </div>
 
-    <footer class="border-top footer text-muted">
+    @*FOOTER*@
+    <footer class="text-center py-5 bg-dark text-white">
         <div class="container">
-            &copy; 2022 - CrowdFundingApplication - <a asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
+            <div class="row">
+                <div class="col-12">
+                    <div class="mb-0">
+                        <a href="#">Github</a>
+                        <a href="#">Linkedin</a>
+                        <a href="#">Facebook</a>
+                        <a href="#">Instagram</a>
+                    </div>
+                    <p class="mb-0 py-3">&copy; 2022 - Design by Nikita. All rights reserved. </p>
```

Comment:
Why is there a space after the dot?

Reply:
Has been deleted the empty space after the dot